stack_configuration.go

```diff
@@ -20,6 +20,16 @@ type StackConfigurations interface {
 
 	// JSONSchemas returns a byte slice of the JSON schema for the stack configuration.
 	JSONSchemas(ctx context.Context, stackConfigurationID string) ([]byte, error)
+
```

Comment:
Can we also add an AwaitQueued? This would be helpful since only then the eventstream url is set :)

Reply:
I added `AwaitStatus` that accepts a specific status, and renamed "AwaitPrepared" to "AwaitCompleted"